ebean-core/src/main/java/io/ebeaninternal/server/deploy/BeanPropertyAssocOne.java

```diff
@@ -811,16 +811,22 @@ public void jsonWrite(SpiJsonWriter writeJson, EntityBean bean) throws IOExcepti
   @Override
   public void jsonRead(SpiJsonReader readJson, EntityBean bean) throws IOException {
     if (jsonDeserialize && targetDescriptor != null) {
-      T target = (T) value(bean);
+      // CHECKME: may we skip reading the object from the json stream?
```

Comment:
I have not changed the behaviour here, but "BeanProperty" will always read the value from the json stream (and discards it, when `jsonDeserialize==false`)
Here we do not consume the bean from the jsonReader